src/main/java/org/opentripplanner/model/plan/Place.java

```diff
@@ -143,7 +143,7 @@ public static Place normal(Vertex vertex, String name) {
 
     public static Place forStop(StopLocation stop, Integer stopIndex, Integer stopSequence) {
         return new Place(
-                stop.getName(),
+                stop.getName().toString(),
```

Comment:
These should use the locale parameter in the routing request

Reply:
Fixed.